src/dataexplorer/dataexplorer-application/src/components/Patient/Patient.tsx

```diff
@@ -18,13 +19,18 @@ const Patient = () => {
         setSchemanumber('');
         setComparableSchemaNumbers([]);
     }, [setSchemanumber, setComparableSchemaNumbers]);
+    const history = useHistory();
 
     useEffect(() => {
         if (patientData && patientData.total === 1) {
             const name = `${patientData?.entry[0].resource.name[0].given[0]} ${patientData?.entry[0].resource.name[0].family}`;
             setName(name);
         }
     }, [patientData, setName]);
+    if (patientData.entry === undefined) {
+        history.push('/');
+        return <></>;
```

Comment:
Possible to just return empty? or not have a return at all?
```suggestion
        return;
```

Reply:
We need a return call (or something with a similar function) in order to stop the rest of the component to render (and thus crashing if patientData.entry is undefined). Since this component is expected to return a JSX element, an empty return function gives a typescript error.